lua/bufferline/render.lua

```diff
@@ -231,13 +231,7 @@ local function render()
   end
 
   -- To prevent the expansion of the last click group
-  result = result .. '%0@BufferlineMainClickHandler@'
-
-  if layout.actual_width + 1 <= layout.buffers_width and len(items) > 0 then
-    local separatorPrefix = hl('BufferInactiveSign')
-    local separator = icons.bufferline_separator_inactive
-    result = result .. separatorPrefix .. separator
-  end
+  result = result .. '%0@BufferlineMainClickHandler@' .. hl('TabLineFill') .. icons.bufferline_separator_inactive
```

Comment:
This latest commit moves the `TabLineFill` to between the tab count and the buffers. As such we don't need `BufferTabpageSpacer` anymore, and can just use the builtin highlight group which most colorschemes define anyway.

Reply:
Thanks for pushing me to find this solution, by the way :+1:

Reply:
Cool! One thing though, if we're going to use `TabLineFill` directly it needs to be documented in `highlights.vim`, in the `doc/barbar.txt` and in the readme, so people know how to style barbar.

Reply:
Will do!